doc/user/content/get-started/quickstart.md

```diff
@@ -13,146 +13,43 @@ aliases:
+When you're ready, head over to the [Materialize console](https://console.materialize.com/),
+and pop open the SQL Shell.
 
-1. Use the [`CREATE CLUSTER`](/sql/create-cluster/) command to create two new clusters:
+## Step 1.
 
-    ```sql
-    CREATE CLUSTER ingest_qck SIZE = '2xsmall';
-    ```
-
-    ```sql
-    CREATE CLUSTER compute_qck SIZE = '2xsmall';
-    ```
+You'll use a sample auction house data set to build an operational use
+case around auctions, bidders and (gasp) fraud. 🦹
 
-    The `2xsmall` size is sufficient for the data ingestion and computation in this getting started scenario.
+As the auction house operator, you want to detect fraudulent behavior as soon as
+it happens, so you can act on it immediately. Lately, you've been struggling
+with [auction flippers](https://en.wikipedia.org/wiki/Flipping) — users that
+purchase items only to quickly resell them for profit.
 
-1. The physical compute resources in your clusters are called [`replicas`](/get-started/key-concepts/#cluster-replicas). Use the [`SHOW CLUSTER REPLICAS`](https://materialize.com/docs/sql/show-cluster-replicas/) command to check the status of the replicas:
-
-    ```sql
-    SHOW CLUSTER REPLICAS WHERE cluster IN ('compute_qck', 'ingest_qck');
-    ```
-    <p></p>
-
-    ```noftm
-       cluster   | replica |  size   | ready
-    -------------+---------+---------+-------
-     compute_qck | r1      | 2xsmall | t
-     ingest_qck  | r1      | 2xsmall | t
-    (2 rows)
-    ```
-
-    Once both replicas are ready (`ready=t`), move on to the next step.
-
-## Step 3. Ingest streaming data
-
-Materialize supports streaming data from multiple external sources, including [Kafka](/sql/create-source/kafka/) and [PostgreSQL](/sql/create-source/postgres/). The process for integrating a source typically involves configuring the source's network and creating connection objects in Materialize.
-
-For this guide, you'll use a [built-in load generator](/sql/create-source/load-generator/#auction) that simulates an auction house, where users bid on an ongoing series of auctions.
-
-1. Most objects in Materialize are [namespaced](/sql/namespaces/) by database and schema, including sources, so start by creating a unique schema within the default `materialize` database:
-
-    ```sql
-    CREATE SCHEMA qck;
-    ```
-
-    ```sql
-    SET schema = qck;
-    ```
-
-1. Use the [`CREATE SOURCE`](/sql/create-source/load-generator/) command to create the auction house source:
+1. Let's start by kicking off the built-in [auction load generator](/sql/create-source/load-generator/#auction),
+   so you have some data to work with.
 
     ```sql
     CREATE SOURCE auction_house
-      IN CLUSTER ingest_qck
-      FROM LOAD GENERATOR AUCTION
-      (TICK INTERVAL '500ms')
-      FOR ALL TABLES;
+    FROM LOAD GENERATOR AUCTION
+    (TICK INTERVAL '1s')
+    FOR ALL TABLES
```

Comment:
```suggestion
```

We removed `For all TABLES` from loadgen sources.

Reply:
We removed `FOR TABLES`; `FOR ALL TABLES` is still a required clause. This made me notice the clause wasn't removed from the documentation, though!

Reply:
Patched the docs in [519e453a8916cdbc6472004a366a041ba46e93f2](https://github.com/MaterializeInc/materialize/pull/21656/commits/519e453a8916cdbc6472004a366a041ba46e93f2).